src/decluttering.md

```diff
@@ -1,9 +1,23 @@
 # Reducing UI clutter and cognitive load
 
-A user interface with too many elements requires more processing from users' brains. When fewer things on the screen try to attract the user's attention, it becomes easier for the user to "consume" the UI. This is what we mean when we talk about "reducing cognitive load". In the end, our goal is for the user to spend their mental energy on the problem at hand, not at the tool they are using to solve it (our UI).
+## The philosophy | Minimalism
+From our point of view, a great product or tool is essentially one that has a minimalistic user interface. This does not limit to digital products, but it becomes extra important as they are often packed with tools, features, and functionalities.
 
-## Buttons
+Both given the aesthetics and functionality concerns, a minimalist product should be reduced to its necessary elements. This can be achieved by usage of fewer design elements, less ornamental elements in its visual design, fewer options and features, and less occupied screen space; as well as carefully breaking the interaction flows or functionalities into distinct and meaningful parts.
 
+Thus when designin a product, we have to keep removing elements and questioning their existence in the UI. We should always keep asking questions such as: Is this really needed? Does this really have to be there? Why? What's the trade-off if it's removed? Do we lose functionality? Do we lose usability? And so forth
+> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.
+>
+> – Antoine de Saint-Exupéry
```

Comment:
Haha! As I was reading this, I was thinking of this quote, and that it would fit nicely in this section. Then I got to this 👌 🤣

Reply:
😅